index.js

```diff
@@ -249,7 +248,7 @@ module.exports = {
             }
 
             return type;
-          }, null)
+          }, null) || 'error'
```

Comment:
Why is this change necessary?

Reply:
Otherwise, the `all` configuration contains `null` value for rules not present in the presets.